ozon/products.go

```diff
@@ -1558,6 +1558,92 @@ func (c Products) GetDescriptionOfProduct(ctx context.Context, params *GetDescri
 	return resp, nil
 }
 
+type GetDescriptionOfProductFilterV4 struct {
```

Comment:
Let's remove any mention of version in naming

Reply:
done